docs/VerifStrat/source/quick_start.rst

```diff
@@ -33,15 +33,32 @@ You will need:
 
 #. A Linux machine (CORE-V-VERIF has been successfully run under Ubuntu, Debian and CentOS).
 #. Python3 and a set of plug-ins. The current plug-in list is kept in `$CORE_V_VERIF/bin/requirements.txt`.
-#. A GCC cross-compiler (aka "the `Toolchain <https://github.com/openhwgroup/core-v-verif/blob/master/mk/TOOLCHAIN.md#core-v-toolchain>`_"). Even if you already have a RISC-V toolchain, please do follow that link and read **TOOLCHAIN.md** for recommended ENV variables to point to it.
+#. A GCC cross-compiler (aka "the `Toolchain <https://github.com/openhwgroup/core-v-verif/blob/master/mk/TOOLCHAIN.md#core-v-toolchain>`_"). Even if you already have a RISC-V toolchain, please do follow that link and read `TOOLCHAIN.md <https://github.com/openhwgroup/core-v-verif/blob/master/mk/TOOLCHAIN.md>`_ for recommended ENV variables to point to it.
 #. `Verilator <https://veripool.org/guide/latest/install.html>`_.
 
-An easy way to get the Python plug-ins installed on your machine is::
+There are two ways to get the Python plug-ins installed. The first one, which installs them in the host machine, is::
 
    $ git clone https://github.com/openhwgroup/core-v-verif.git $CORE_V_VERIF
    $ cd $CORE_V_VERIF/bin
    $ pip3 install -r requirements.txt
 
+The second one, which is more complex and more robust, is to create a
+Python virtual environment and install them there::
+
+   $ git clone https://github.com/openhwgroup/core-v-verif.git $CORE_V_VERIF
+   $ cd $CORE_V_VERIF/bin
+   $ pip3 install virtualenv
+   $ python3 -m venv $VENV_NAME
+   $ source $(VENV_NAME)/bin/activate
+   $ pip3 install -r requirements.txt
+
+Before running the simulation the following environment variables
```

Comment:
This is technically correct information but is already documented elsewhere.  It is good to follow the "DRY principle" of documentation: `Don't Repeat Yourself`.  This information is already documented in [mk/README](https://github.com/openhwgroup/core-v-verif/tree/master/mk#required-corev-environment-variables), so I would add a bullet item to the "You will need" list above.  Something like:

- Set the required shell environment variables ([link](https://github.com/openhwgroup/core-v-verif/tree/master/mk#required-corev-environment-variables)).

Reply:
I totally agree with this. I like sentence you propose so I will change it👍